### PR TITLE
added way to rename branch

### PR DESCRIPTION
**Pull request by ujwaldhakal**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- [x] Added method to rename GitHub branch locally and remotely

## Usage
`grename oldbranch newbranch`

### CODE REVIEW

**Comment by larson-carter** [comment]
@ujwaldhakal Can you please follow the Proper Pull Request template that we have setup. It makes the process of reviewing PRs so much easier and quicker. Otherwise PRs that have followed the proper layout will get looked at first.

**Comment by ujwaldhakal** [comment]
hi @larson-carter I have re-implemented your default PR layout can you please review and tell if it's ok

File: plugins/git/git.plugin.zsh [hunk 1]
```diff
@@ -255,3 +255,21 @@ alias glum='git pull upstream master'
...
+  then
+    echo "use command like 'grename old_branch new_branch'"
+    return
+  fi
+
+  if [ -z "$2" ]
+  then
+    echo "use command like 'grename old_branch new_branch'"
+    return
+  fi
```
Comment (mcornella) [inline plugins/git/git.plugin.zsh]: This check can be added to the previous if

File: plugins/git/git.plugin.zsh [hunk 2]
```diff
@@ -255,3 +255,21 @@ alias glum='git pull upstream master'
 
 alias gwch='git whatchanged -p --abbrev-commit --pretty=medium'
 alias gwip='git add -A; git rm $(git ls-files --deleted) 2> /dev/null; git commit --no-verify --no-gpg-sign -m "--wip-- [skip ci]"'
+
+function grename() {
+  if [ -z "$1" ]
+  then
+    echo "use command like 'grename old_branch new_branch'"
```
Comment (mcornella) [inline plugins/git/git.plugin.zsh]: The message here could be better, i.e. follow common conventions. This is better:
```zsh
echo "Usage: $0 old_branch new_branch"
```
Also, use `return 1` to indicate a failure to run the command.

File: plugins/git/git.plugin.zsh (hunk 1 above)
Comment (ujwaldhakal) [reply plugins/git/git.plugin.zsh]: I have added next if on a single line

File: plugins/git/git.plugin.zsh (hunk 2 above)
Comment (ujwaldhakal) [reply plugins/git/git.plugin.zsh]: Thank you for review. I have done some changes please let me know if there is anything i would need to do

**Pull request merged**